__tests__/src/message.js

```diff
@@ -0,0 +1,18 @@
+
+import message, {setMessages} from '../../src/message'
+
+describe('message', () => {
+  const AUTO_SLUGGED_MESSAGE = 's-l$U/g*g.e d'
+  const DEFAULT_MESSAGE = 'default message'
+  setMessages({one: 'hello world', two: 'hola %s', 's-l-u-g-g-e-d': 'cool'})
+
+  it('should work consistently', () => {
```

Comment:
I think it would be good to separate these into different test cases for each message type.  Otherwise the test fails for any of them.